metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -674,6 +781,21 @@ def gen_dataset_workunits(
                 view_properties_aspect,
             )
 
+    def gen_tag_workunits(self, tag: SnowflakeTag) -> Iterable[MetadataWorkUnit]:
+        tag_key = str(tag)
+        tag_urn = make_tag_urn(tag_key)
+
+        tag_properties_aspect = TagProperties(
+            name=tag_key,
+            description=f"Represents the Snowflake tag `{tag_key.split('=')[0]}` with value `{tag.value}`.",
```

Comment:
Maybe add a `from_str` in SnowflakeTag class and use it here to abstract out the tag string formation ?